### PR TITLE
Adding new dropdown class for descriptive dropdowns

### PR DESCRIPTION
**Pull request by kwalker3690**
Adds #352

@danoc @tomgenoni

### CODE REVIEW

File: src/oui/partials/components/_dropdown.scss [hunk 1]
```diff
@@ -114,4 +136,23 @@
...
+  &--descriptive {
+    .#{$namespace}dropdown__item {
+      white-space: normal;
+    }
+
+    .#{$namespace}dropdown__block-link {
+      cursor: pointer;
+    }
+
+    &__header {
```
Comment (danoc) [inline src/oui/partials/components/_dropdown.scss]: Should these all have the namespace thing?

Comment (kwalker3690) [inline src/oui/partials/components/_dropdown.scss]: No because they are nested

Comment (kwalker3690) [inline src/oui/partials/components/_dropdown.scss]: They inherit the namespace from above

File: src/oui/partials/components/_dropdown.scss [hunk 2]
```diff
@@ -114,4 +136,23 @@
     right: 0;
     left: auto;
   }
+
+  &--descriptive {
+    .#{$namespace}dropdown__item {
+      white-space: normal;
+    }
+
+    .#{$namespace}dropdown__block-link {
```
Comment (danoc) [inline src/oui/partials/components/_dropdown.scss]: This should probably be applied to all `/dropdown__block-link`, not just within `.dropdown--descriptive`.

**Pull request merged**